main_routers/characters_router.py

```
@@ -0,0 +1,706 @@
+    
+    # 检查当前角色是否有活跃的语音session
+    if old_catgirl and old_catgirl in session_manager:
+        mgr = session_manager[old_catgirl]
+        if mgr.is_active:
+            from main_logic.omni_realtime_client import OmniRealtimeClient
+            is_voice_mode = mgr.session and isinstance(mgr.session, OmniRealtimeClient)
+            
+            if is_voice_mode:
+                return JSONResponse({
+                    'success': False, 
+                    'error': '语音状态下无法切换角色，请先停止语音对话后再切换'
```

Ruff (0.14.8)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)